addon/components/sortable-group.js

```diff
@@ -18,29 +19,35 @@ export default Component.extend({
     @property items
     @type Ember.NativeArray
   */
-  items: computed(() => { return a(); }),
+  items: computed({
+    get() { return a(); }
```

Comment:
I'll go back to my app and confirm where the warning came from - but if definitely was related to computed properties that are provided a function rather than an object. 

I believe it was in the `y` computed property..